### PR TITLE
Ft setup react file

### PR DESCRIPTION
**Pull request by Maxcutex**
setup up react test 
rearrange folders in react project 
setup home page components static pages.

### CODE REVIEW

File: client/src/__tests__/Home/FeaturedSection.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  //     var node = component.find("nav");
+  //     expect(node.length).toEqual(1);
+  //   });
+  //   it("has shows two small section component", () => {
+  //     const component = shallow(<Header />);
+  //     var node = component.find("nav");
+
+  //     expect(node.length).toEqual(1);
+  //   });
+});
```
Comment (Charpell) [inline client/src/__tests__/Home/FeaturedSection.test.js:27]: Remove commented lines of code

File: client/src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+import "../App.css";
+import "../assets/css/style.css";
+import "../assets/css/bootstrap.css";
+// import Loader from "./Loader/Loader";
```
Comment (Charpell) [inline client/src/components/App.js:5]: Remove comments

File: client/src/index.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import "./index.css";
-import App from "./App";
+import App from "./components/App";
 // import * as serviceWorker from './serviceWorker';
```
Comment (Charpell) [inline client/src/index.js:6]: Remove comments

**Review by Charpell — COMMENTED** [review]
Great job @Maxcutex for this work. However, take note of my feedback

The PR is too large, I feel you should have broken it down to different stories. The react setup, the UI, the test setup and individual test should have different stories.
I think the client side of this app should be in another repository. 
The Readme does not contain installation guide, there is no doc showing me how I could test the app in my browser.

File: client/src/index.js (hunk 3 above)
Comment (Maxcutex) [reply client/src/index.js:6]: ok will do .. am following a school of thought that allows the react + ruby to sit in the same app folder

Comment (Charpell) [reply client/src/index.js:6]: Do you have the instance of the ruby hosted? This will allow me to test the react repo independently?

**Pull request merged**